Jenkinsfile

```diff
@@ -150,7 +150,34 @@ pipeline {
               }
             }
           }
-        } 
+        }
+        stage('TPU VM') {
```

Comment:
Given that this image is built against a fixed version of the CPU image (here, v108), I don't think we should rebuild it all the time. Or is it the goal to not pin it to v108 eventually?